### DISCUSSION TITLE
Can I use imagesharp to implement such a task?

### DISCUSSION THREAD

**Discussion by sadness-mugetsu** [SixLabors/ImageSharp#1317] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68119409/90403994-d1fc6100-e0b2-11ea-8896-fe01309c58d6.png)
I am learning to write a discord of bots, I managed to make the bot greet the user on a separate channel with a normal message, can I implement using imagesharp to display the avatar of the user who just logged on to the server in the center as shown in the example of the photo, I use the library to develop the bot discord.net

**Comment by tocsoft** [comment, marked as answer]
you should checkout out samples repo. https://github.com/SixLabors/Samples

Specifically https://github.com/SixLabors/Samples/blob/master/ImageSharp/AvatarWithRoundedCorner/Program.cs which is an example of drawing an avatar with rounded corners. 

Alternatively you could use an `ImageBrush` and fill in an `EllipsePolygon` with the image (for make it render correctly ensure the image has been resized to fit the final circle).
